tests/test_download_image.py

```diff
@@ -41,5 +41,5 @@ def test_download_image(mock_get):
     font_path = "app/static/fonts/Arial.ttf"
     font_size = 20
     font = ImageFont.truetype(font_path, font_size)
-    text_bbox = draw.textbbox((10, 10), "Test", font=font)
+    text_bbox = draw.textbbox((10, 20), "Test", font=font)  # Adjusted y-coordinate
```

Comment:
Do we need this?

Reply:
No, we don't need this adjustment in the test. The adjustment was initially added to match the backend logic, but since we're now focusing on logging to identify the root cause, we should revert this change to keep the test consistent with the actual functionality.

I'll update the PR to remove this adjustment. Thank you for pointing it out!